Algorithm.Framework/Execution/StandardDeviationExecutionModel.py

```diff
@@ -118,9 +118,8 @@ def __init__(self, algorithm, security, period, resolution):
         algorithm.RegisterIndicator(symbol, self.STD, self.Consolidator)
 
         # warmup our indicators by pushing history through the indicators
-        history = algorithm.History(symbol, period, resolution)
-        if 'close' in history:
-            history = history.close.unstack(0).squeeze()
-            for time, value in history.iteritems():
-                self.SMA.Update(time, value)
-                self.STD.Update(time, value)
+        data_type = QuoteBar if security.Type == SecurityType.Forex or security.Type == SecurityType.Cfd else TradeBar
```

Comment:
This line feels a bit fragile, maybe more generically can replace `data_type =` for `Consolidator.InputType` so it's not hard coded?